### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by Arboghast**
Some of my unit testing and java code spilled over into this CL, my bad, just ignore it for now and they will get their own CL's soon. I made a lot of css changes with responsiveness in mind aswell as javascript function optimizations and abstractions. As for the website, It looks like the deployed site defaults to the Master branch(hence the ios issues @taylorwan ). I don't think taking screenshots is the best way to show responsiveness but here is what it currently looks like. After this CL is merged I'll send you both the link to the site for some feedback.

Screenshot 2020-07-14 at 4 35 25 PM - Display 1
Screenshot 2020-07-14 at 4 35 17 PM - Display 1
Screenshot 2020-07-14 at 4 34 48 PM - Display 1
Screenshot 2020-07-14 at 4 34 24 PM - Display 1
Screenshot 2020-07-14 at 4 34 16 PM - Display 1

### CODE REVIEW

**Comment by mithasgoogle** [comment]
238 file commit!  Can we add some of these to a .gitignore or draw more attention to specific files that need review?  This is a lot of process.

**Comment by mithasgoogle** [comment]
Can you just point me to the files that need review?  Hard to dig through 200+ to figure out which ones to focus on.

**Comment by Arboghast** [comment]
> Can you just point me to the files that need review? Hard to dig through 200+ to figure out which ones to focus on.

Just this file for now, I'll ask for some feedback on the design and UI once I get the site deployed.
https://github.com/Arboghast/Portfolio/pull/6/files#diff-727098be44d5aeada596189d6fab7912

File: portfolio/src/main/webapp/js/comments.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+ *
+ * Generates comment html using the recieved json and
+ * a delete button to remove each comment via id
+ */
+function getComments() {
+    const limitInput = document.getElementById('comment-limit');
+    const value = limitInput.options[limitInput.selectedIndex].value;
+  
+    let blogTitle = document.getElementById('blog-title').innerHTML;
+    blogTitle = blogTitle.split(' ').join('_'); // Clean the url
```
Comment (mithasgoogle) [inline portfolio/src/main/webapp/js/comments.js:13]: Nit: can you provide an example of how the URL would be cleaned by this statement?  I.e., "Before: xxx.com --> After: xxx"

File: portfolio/src/main/webapp/js/comments.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    if (value != 'null') {
+      queryString += '&limit=' + value;
+    }
+  
+    const languageInput = document.getElementById('comment-language');
+    const language = languageInput.options[languageInput.selectedIndex].value;
+    queryString += '&language=' + language;
+  
+  
+    fetch(queryString)
```
Comment (mithasgoogle) [inline portfolio/src/main/webapp/js/comments.js:25]: 1. Comment what's going on in this block of code.
2. Consider creating a function that handles the work you do if err is caught.

**Pull request merged**